pants-plugins/pants_explorer/server/graphql/query/targets.py

```diff
@@ -63,7 +63,7 @@ class Target:
     target_type: str = strawberry.field(
         description="The target type, such as `python_sources` or `pex_binary` etc."
     )
-    fields: JSONScalar = strawberry.field(
+    fields: JSONScalar = strawberry.field( # type: ignore[valid-type]
```

Comment:
This has been a roller coaster. Was added during the upgrade to Python 3.11, then I removed it a day or two ago as Mypy warned about it.

Now, on upgraded mypy, it's back to being a problem (it also fails pyright).

There are a bunch of open tickets regarding strawberry's typing of fields/decorators - so this file is basically just all red squiggles, and mypy seems to be chill enough about it.